react/SearchAction.tsx

```diff
@@ -22,7 +22,7 @@ function SearchAction({ searchTermPath }: Props) {
           url: baseUrl,
           potentialAction: {
             '@type': 'SearchAction',
-            target: `${baseUrl}${path}{search_term_string}`,
+            target: `${baseUrl}${path}{search_term_string}?map=ft`,
```

Comment:
@hiagolcm @thalytafabrine, maybe you have more context about the search... Can you take a look here?